### ISSUE TITLE
Build error:  'Document' is not exported from 'flexsearch'

### ISSUE THREAD

**Issue by tarjelavik** [samvera-labs/clover-iiif#193]
Hi, tried the the version 2.7.1 and 2.7.2 in a Nextra app and got this error when building:

```bash
./node_modules/@samvera/clover-iiif/dist/scroll/index.mjs
Attempted import error: 'Document' is not exported from 'flexsearch' (imported as 'Q').

Import trace for requested module:
./node_modules/@samvera/clover-iiif/dist/scroll/index.mjs
./node_modules/@samvera/clover-iiif/dist/index.mjs
./components/work.tsx
```

The components is basically the demo component.

**Comment by mathewjordan** [comment]
Apologies for this @tarjelavik and thank you for submitting this. This should now be resolved by 2.7.3. Can you take a look and confirm on your end?

**Comment by tarjelavik** [comment]
No worries :-). Yes, i can confirm that 2.7.3 works and does not break the Nextra build. 

Thank you for the quick fix!
